disco_aws_automation/disco_vpc.py

```diff
@@ -451,7 +451,7 @@ def _destroy():
                 self.boto3_ec2.delete_network_interface(NetworkInterfaceId=interface['NetworkInterfaceId'])
 
         # Keep trying because delete could fail for reasons based on interface's state
-        keep_trying(300, _destroy)
+        keep_trying(600, _destroy)
```

Comment:
We can experiment with detaching the network interfaces before trying to delete them. But I think @IlyaSukhanov had some concerns with that approach.